subsys/net/ip/net_pkt.c

```diff
@@ -550,10 +550,10 @@ static struct net_pkt *net_pkt_get(struct k_mem_slab *slab,
 #endif /* CONFIG_NET_CONTEXT_PRIORITY */
```

Comment:
In commit "net: pkt: Set default data_len to iface MTU", the commit message is hard to understand. Perhaps something like this could be used:

`When creating net_pkt, the default value of net_pkt data_len was set to zero. In case of RAW sockets, when family is AF_PACKET, the data_len will be zero as there is no higher level protocol information. So in this case, the default value of data_len must be set to interface MTU.`

Reply:
ok